### ISSUE TITLE
Cannot able to set focus on "Find out more about the Xamarin Workbooks Experience Improvement Program" hyperlink in Feedback tab of preferences

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#373]
### Steps to Reproduce

1. Launch Xamarin Workbooks application.This will invokes "Create a new Workbook" window.
2. Navigate to Tools -> Preferences and invoke it. This will invoke general tab of preferences window.
3. Invoke Feedback tab using Mouse (As it is not accessible by keyboard).
4. Navigate to "Find out more about the Xamarin Workbooks Experience Improvement Program" link and set the keyboard focus

### Expected Behavior
Should be able to set keyboard focus on "Find out more about the Xamarin Workbooks Experience Improvement Program" hyperlink.
### Actual Behavior
Cannot able to set focus on "Find out more about the Xamarin Workbooks Experience Improvement Program" link in Feedback tab of preferences.

After Pressing tab from Feedback, loss of focus is observed and for next tab press focus goes to "Yes, I am willing to Participate (Recommended)" radio button.

Inspect Behavior:
After Pressing Tab from "Feedback Tab" item focus goes to content area "Help Improve Xamarin Workbooks. Please refer below screenshot.
![screenshot 174](https://user-images.githubusercontent.com/22956130/40759531-1765ca46-64b0-11e8-8162-507eb247961b.png)

### User Impact:
Keyboard user cannot access the hyperlink.
### 
![issue 7](https://user-images.githubusercontent.com/22956130/40759632-9218405c-64b0-11e8-8e9c-23af54b8e576.png)
Screenshot:

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-MAC
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
